### PR TITLE
Chat UI Library Quickstarts

### PR DESCRIPTION
**Pull request by dhiraj**
## Purpose

* Add quick start sample apps for Java and Kotlin
* Reorganize `ui-library-quick-starts` folder to house both calling and chat quick starts, allowing for future quick start additions

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Publish uilibrary-chat to local maven repository
* Fill in the chat variables in MainActivity and run the app

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

## What to Check
Verify that the following are valid
* Pressing the Start Experience button should start and join the chat thread
* Chat operations should work as expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,5 +10,18 @@ Azure Communication Services enable developers to add communication capabilities
...
+2. [Add video calling to your Android App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-video-calling)
+3. [Add video calling to your WebView Android App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-android-webview)
+4. [Add telephone calling to your Android App](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/pstn-call?pivots=platform-android)
+5. [Record Calls](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-call-recording)
+6. [Handle Calling Events](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/handle-calling-events)
+7. [Raw media Access](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-raw-media-access)
+
+### Chat SDK Quickstarts
+1. [Add Chat to our App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/get-started)
```
Comment (Leah-Xia-Microsoft) [inline README.md]: Add Chat to **y**our App

File: README.md [hunk 2]
```diff
@@ -10,5 +10,18 @@ Azure Communication Services enable developers to add communication capabilities
...
+2. [Using Chat SDK with React Native](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/react-native)
+
+### UI Libraries
+1. [Getting Started with the UI Libraries](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/ui-library/get-started-composites?tabs=kotlin&pivots=platform-android)
```
Comment (Leah-Xia-Microsoft) [inline README.md:27]: The url will change once we publish the MSdoc. It will be `get-started-calling-ui-library` instead of `get-started-composites`

Comment (pavelprystinka) [reply README.md:27]: @Leah-Xia-Microsoft name of the page is changed in left hand menu, but URL remain the same (for now)

File: README.md (hunk 1 above)
Comment (pavelprystinka) [reply README.md]: Fixed

**Review by iaulakh — APPROVED** [review]

**Pull request merged**